lib/disk.js

```diff
@@ -40,8 +48,18 @@ class DiskWrapper {
 			case constants.E2FS_BLK_DISCARD:
 				promiseToCallback(this.disk.discard(offset, length), callback);
 				break;
+			case constants.E2FS_BLK_ZEROOUT:
+				// Writes zeroes then discards, TODO: test case
```

Comment:
Does this still need a test case?

Reply:
We have no use case for this and I don't know how to "ask" ext2fs to issue such request.

Reply:
Aye, okay